src/meshcat/commands.py

```diff
@@ -44,12 +44,35 @@ def lower(self):
             u"matrix": list(self.matrix.T.flatten())
         }
 
-class CaptureImage:
+
+class SetCamTarget:
+    __slots__ = ["value"]
+    def __init__(self, value):
```

Comment:
`value` is pretty vague as a variable name. What are the expected contents, and what do they do?

Reply:
It's supposed to be the camera's target position, in 3D. I made this more explicit the commit I pushed just now.